src/lake/lake.cpp

```diff
@@ -2,8 +2,12 @@
 #include "duckdb/common/unordered_set.hpp"
 #include "rust_extensions/delta.hpp"
 
+#include <utility>
+
```

Comment:
remove?

Reply:
It's needed for `std::move`; if we enable IWYU it will complain on missing headers.

Reply:
I think duckdb already implements quite a lot of STL containers and utils, but seems `<utility>` is still needed.
For example, https://github.com/duckdb/duckdb/blob/448b46937fcafdc789b053536d093ef22b9105b5/src/common/types/value.cpp#L36